ironfish-cli/src/commands/deposit-all.ts

```diff
@@ -224,6 +226,14 @@ export default class DepositAll extends IronfishCommand {
           const transaction = result.content
           txs.push(transaction)
         } catch (error) {
+          if (
+            error instanceof RpcRequestError &&
+            error.code === ERROR_CODES.INSUFFICIENT_BALANCE
+          ) {
+            // Our balance changed while trying to create a payout, ignore this error
+            await PromiseUtils.sleep(30000)
+          }
+
```

Comment:
Does this need a `continue` or an `else` block? It looks like this will sleep 30s but then throw the error

Reply:
Good catch, thank you I forgot to bring that over from my other branch.